### PR TITLE
Adding Billin partners ES page

### PR DESCRIPTION
**Pull request by jamesshedden**
gocardless para billin gocardless

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

File: app/pages/partners/billin/partners-billin.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+                  <p className='u-text-heading u-color-dark-gray u-text-m u-text-light u-margin-Txxs u-text-no-smoothing'>
+                    Gestiona el proceso de facturación con Billin<br/>
+                    y garantiza el pago de tus facturas<br/>
+                    a tiempo con GoCardless
+                  </p>
+
+                  <div className='u-margin-Tl'>
+                    <a href='https://www.billin.net/programa-facturacion-domiciliacion-bancaria/?utm_source=Gocardless&utm_medium=web&utm_content=billin&utm_campaign=billin'
+                    id='track-cta-pricing' className='btn u-pull-start'>
+                      Get started
```
Comment (jamesshedden) [inline app/pages/partners/billin/partners-billin.js]: @AlfonsoSB Need a translation here for `Get started`

File: app/pages/partners/billin/partners-billin.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+        <hr className='section-divider' />
+
+        <div className='site-container u-text-center u-padding-Vxxl'>
+          <div className='grid u-padding-Txl u-padding-Bm'>
+            <div className='grid__cell u-size-1of2 u-padding-Rm'>
+              <div className='numbered-step u-margin-Bm'>1</div>
+
+              <h2 className='u-text-l u-text-heading u-color-dark-gray u-text-light u-margin-Bm'>
+                Connect your account
```
Comment (jamesshedden) [inline app/pages/partners/billin/partners-billin.js]: @AlfonsoSB Are you able to adapt your content for this section, for steps 1 - 4? Each one is made up of a simple heading of ~3 words that must fit on one line eg `Connect your account` and then a short paragraph below

File: app/pages/partners/billin/partners-billin.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+              a cobrar inmediatamente
+              </h2>
+
+              <p className='u-text-xs u-color-dark-gray u-margin-Vxs'>
+                Billin y la integración con GoCardless son<br/>completamente gratuitos. Sin costes de setup,<br/>mensualidades ni comisiones ocultas.
+              </p>
+
+              <a href='https://www.billin.net/programa-facturacion-domiciliacion-bancaria/?utm_source=Gocardless&utm_medium=web&utm_content=billin&utm_campaign=billin'
+              to='pricing' id='track-cta-pricing' className='btn btn--hollow u-margin-Tm'>
+                Start using QuickBooks
```
Comment (jamesshedden) [inline app/pages/partners/billin/partners-billin.js]: @AlfonsoSB Be great to get a translation here for `Start using Billin`

**Comment by jamesshedden** [comment]
@AlfonsoSB Hey - be awesome to get some help from you to finish the content for this page! I've left a few notes where I need particular assistance from you with translated content.

In the end, the screenshots I managed to grab from your original Google Doc worked OK for the screenshots. If Billin manage to send us some higher resolution ones, that'd still be great, but in the meantime the above should be OK for now.

I've left out the quote section for now - we didn't have this yet in the QuickBooks page, so I didn't have anything to copy over. I can code it but will just take a bit of extra time and wanted to get this up as soon as possible as I know we communicated a time expectation to the partner! It'd be great to get this ready for merging, and deploy as it is, and deal with the quote later, so there is less time pressure.

Hope all sounds good, let me know if any questions/problems 🌵

File: app/pages/partners/billin/partners-billin.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+                </p>
+              </div>
+
+              <div className='grid__cell u-size-1of3 u-text-center'>
+                <figure className='svg-icon u-center'>
+                  <SaveMoneyGreenIcon className='svg-icon__image' />
+                </figure>
+
+                <div className='u-text-heading u-color-dark-gray u-text-light u-text-no-smoothing u-text-m u-margin-Tm'>
+                  Cobra sin retrasos
```
Comment (jamesshedden) [inline app/pages/partners/billin/partners-billin.js]: @AlfonsoSB I'm not totally sure whether the icons match up to your copy here - what are the English translations of these three headings under the icons?

Comment (AlfonsoSB) [inline app/pages/partners/billin/partners-billin.js]: @jamesshedden 
1. Rápido e intuitivo: Fast and easy to use
2. Ten el control: Have total control
3. Cobra sin retrasos: Get paid on time

File: app/pages/partners/billin/partners-billin.js (hunk 1 above)
Comment (AlfonsoSB) [inline app/pages/partners/billin/partners-billin.js]: "Empieza Ya" @jamesshedden

File: app/pages/partners/billin/partners-billin.js (hunk 3 above)
Comment (AlfonsoSB) [inline app/pages/partners/billin/partners-billin.js]: "Empieza a usar Billin"

**Pull request merged**